openapi/components/schemas/CheckoutForm/CommonCheckoutForm.yaml

```
@@ -13,6 +13,10 @@ properties:
     type: string
     allOf:
       - $ref: ../ResourceId.yaml
+  customDomain:
+    description: Custom domain.
+    type: string
+    nullable: true
   plans:
     description: >-
       The list of plans that would be applied to customer order by default.
```

nitpick: Where possible, avoid future or past tense

```suggestion
      List of plans that are applied to a customer order by default.
```